src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlFieldTrumbowygEditorSettings.Edit.cshtml

```diff
@@ -1,18 +1,16 @@
 @model OrchardCore.ContentFields.ViewModels.TrumbowygSettingsViewModel
 <div id="@Html.IdFor(m => m)" class="field-editor field-editor-trumbowyg">
-    <div class="form-group col-md-6">
+    <div class="form-group w-md-50">
         <div class="custom-control custom-checkbox">
             <input type="checkbox" class="custom-control-input" asp-for="InsertMediaWithUrl" checked="@Model.InsertMediaWithUrl" />
             <label class="custom-control-label" asp-for="InsertMediaWithUrl">@T["Insert Media with url"]</label>
             <span class="hint">— @T["Whether the url is used when selected from Insert Media (By default, the Short Code is inserted)."]</span>
         </div>
     </div>
-    <div class="form-group row">
-        <div class="col-large">
```

Comment:
Do not change this class, it provides a variation of sizing across different screen dimensions. 
That was the point of my change, to return the css to as it was.
I imagine it will be discussed further during the meeting.

Reply:
Do you mean to leave any class such as `col-large`, col-meduim` .. etc?

Reply:
When should we use `col-large` instead of `w-lg-100`?